koku/koku/sentry.py

```diff
@@ -9,13 +9,29 @@
 
 LOG = logging.getLogger(__name__)
 
+BLOCK_LIST = [
+    "/api/cost-management/v1/status/",
+    "/api/cost-management/v1/source-status/",
+]
```

Comment:
Since these values are unique, this can be a set.
```suggestion
BLOCK_LIST = {
    "/api/cost-management/v1/status/",
    "/api/cost-management/v1/source-status/",
}
```